client/src/components/BoulderCard.spec.jsx

```diff
@@ -1,47 +1,68 @@
 import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import BoulderCard from './BoulderCard';
 
 const Boulder = {
-    number: 1,
-    name: 'Black Widow and the people',
-    sector: 'Monkey Island',
-    level: '5',
-    handle_color: 'black',
-    setter: 'Martin Pagels',
-    tags: ['Dyno', 'Sloper'],
-    img_start: 'Boulder-girl.jpg',
-    img_all: 'imageAll.jpg',
-    weighting: 100,
-    number_of_likes: 5,
-    start_date: '2022-04-26',
-    expiry_date: '',
-    climbers: [],
-  }
+  number: 1,
+  name: 'Black Widow and the people',
+  sector: 'Monkey Island',
+  level: '5',
+  handle_color: 'black',
+  setter: 'Martin Pagels',
+  tags: ['Dyno', 'Sloper'],
+  img_start: 'Boulder-girl.jpg',
+  img_all: 'imageAll.jpg',
+  weighting: 100,
+  number_of_likes: 5,
+  start_date: '2022-04-26',
+  expiry_date: '',
+  climbers: [],
+};
 
-  describe('BoulderCard', () => {
-      it('renders the a list item with a title, an img, 5 details and a button', () => {
-          render(
-              <BoulderCard id={Boulder.number} name={Boulder.name} 
-              sector={Boulder.sector} level={Boulder.level} handle_color={Boulder.handle_color}
-              likes = {Boulder.number_of_likes}/>
-          );
-          const cardHeader = screen.getByRole('heading');
-          const cardPicture = screen.getByRole('img');
-          const button = screen.getByRole('button');
+describe('BoulderCard', () => {
+  it('renders the a list item with a title, an img, 5 details and a button and renders four more details after button click', () => {
+    render(
+      <BoulderCard
+        id={Boulder.number}
+        name={Boulder.name}
+        sector={Boulder.sector}
+        level={Boulder.level}
+        handle_color={Boulder.handle_color}
+        likes={Boulder.number_of_likes}
+        tags={Boulder.tags}
+        setter={Boulder.setter}
+        weighting={Boulder.weighting}
+      />
+    );
+    const cardHeader = screen.getByRole('heading');
+    const cardPicture = screen.getByRole('img');
+    const button = screen.getByRole('button');
 
-          const name = screen.getByText(/Black widow and the people/i);
-          const id = screen.getByText(/#1/);
-          const sector = screen.getByText(/Monkey Island/i);
-          const level = screen.getByText(/Level: 5/);
-          const handle_color = screen.getByText(/black/);
+    const name = screen.getByText(/Black widow and the people/i);
```

Comment:
In line 48 you check if `cardHeader` has this text content. I think you don't  need to double check `name`. 🤔